Hymns/Favorites/FavoritedEntity.swift

```diff
@@ -0,0 +1,15 @@
+import SwiftUI
+import RealmSwift
+
+//Realm model of a favorited hymn
+class FavoritedEntity: Object, Identifiable {
```

Comment:
Prefer this style:
/**
 *
*/ 

That way it'll appear in XCode's quick-help

Reply:
Cool!